src/main/java/com/example/animalshelter/model/Animal.java

```diff
@@ -1,25 +1,57 @@
 package com.example.animalshelter.model;
 
-import lombok.Builder;
 import lombok.Data;
 
-import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Positive;
+import javax.persistence.*;
 
 /**
  * Keeps basic information about an animal
  */
 @Data
-@Builder
+@Table
+@Entity
```

Comment:
If you are not adding any values to the `@Table` annotation is profitless - so you can just skip it.
The same goes for the `@Column`. [Here](https://www.baeldung.com/jpa-entities) you can read the basics about those.

Reply:
I have removed @Table and @Column annotations.